runtime-modules/storage/src/lib.rs

```diff
@@ -2826,21 +2827,24 @@ impl<T: Trait> Module<T> {
             Error::<T>::DynamicBagExists
         );
 
-        let bag_change = upload_params.as_ref().map_or(Ok(None), |params| {
-            // check params are valid
-            ensure!(
-                params.expected_data_size_fee == Self::data_object_per_mega_byte_fee(),
-                Error::<T>::DataSizeFeeChanged
-            );
-
-            Self::check_global_uploading_block()?;
-
-            Self::ensure_objects_creation_list_validity(&params.object_creation_list)?;
+        // call can upload data explicitly
+        // add extra tests in the
+        let bag_change = upload_params
+            .as_ref()
+            .map(|params| {
+                // ensure coherent account ids for prize
+                if let Some(deletion_prize) = deletion_prize {
```

Comment:
This check doesn't cover all options for incoherent accounts. A test could highlight it.